uber/models/attendee.py

```diff
@@ -755,20 +756,34 @@ def worked_hours(self):
         return weighted_hours + self.nonshift_hours
 
     def requested(self, department):
-        return department in self.requested_depts_ints
+        return department in self.requested_depts
 
     def assigned_to(self, department):
-        return int(department or 0) in self.assigned_depts_ints
+        return department in self.assigned_depts
```

Comment:
We probably need a version of ``assigned_to`` that lets us look it up by name.  We sometimes configure ``AutomatedEmail``s to do stuff like "email everyone in Tech Ops" e.g. https://github.com/magfest/magprime/blob/master/magprime/automated_emails.py#L50

In that case it would be nice to be able to say something like ``a.assigned_to('Tech Ops')``.  This would require us to check whether we'd been passed a department id, a department instance, or a department name, which might be annoying but which I kind of like the flexibility of.  No strong opinion on that though - the main thing is that we need to be able to configure ``AutomatedEmail`` instances which reference a department in a sensible and easy-to-read way.

Reply:
Hmmm... I had not planned on carrying over the value calculated from the slug, but that may be the easiest stop-gap until we can overhaul the email system as well. If I migrate the calculated value into the database, then we can keep that code unchanged.